src/applications/vaos/services/slot/transformers.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Transforms /vaos/facilities/${facilityId}/available_appointments?type_of_care_id=${typeOfCareId}&clinic_ids[]=${clinicId}&start_date=${startDate}&end_date=${endDate} to
+ * /Slot?status=free
+ *
+ * @export
+ * @param {Array} slots A list of appointment slots from var-resources
+ * @returns {Array} A list of Slot resources in FHIR format
+ */
+export function transformSlots(slots) {
+  return slots.map(slot => ({
+    status: 'free',
```

Comment:
I think this field is `freeBusyType`, rather than `status`.

Reply:
Can you point me to where you see this?  I think that the values should be in the `freeBusyType` set, but that field itself is named `status` according to [the docs](https://www.hl7.org/fhir/slot.html).

Reply:
We have to use the DSTU2 spec, rather than the lates: http://hl7.org/fhir/dstu2/slot.html